app/uk/gov/hmrc/eoricommoncomponent/frontend/services/RegistrationDetailsService.scala

```diff
@@ -48,7 +48,8 @@ class RegistrationDetailsService @Inject() (sessionCache: SessionCache)(implicit
       )
 
       organisationType match {
-        case SoleTrader | Individual | ThirdCountryIndividual | ThirdCountrySoleTrader =>
+        case SoleTrader | Individual | ThirdCountryIndividual | ThirdCountrySoleTrader | IsleOfManSoleTrader |
+            IsleOfManIndividual =>
```

Comment:
This is growing a little silly. Lets have this as a set in CdsOrganisationType (say IndividualOrganisations?) and have an if(CdsOrganisationType.IndividualTraders.contains(organisationType)) instead. Or even a an isIndividualTrader(organisation) def to call like so - if(CdsOrganisationType.isIndividualTrader(organisation))